Hunting Queries/CustomLogs/AWSBucketAPILogs-SuspiciousAcesstoS3BucketsfromValidAccounts.yaml

```diff
@@ -3,6 +3,8 @@ name: Suspicious access to S3 buckets from Valid Accounts
 description: |
   Adversaries may use existing IAM role or users to access private S3 buckets and do malicious actions. This query will look for any access with Assume Role operations which was not historically seen.
```

Comment:
needs single quotes around just in case characters get updated

Reply:
updated.